File: app/src/main/java/com/babestudios/companyinfouk/data/model/persons/Persons.java
package com.babestudios.companyinfouk.data.model.persons;


import com.google.gson.annotations.SerializedName;

import java.util.ArrayList;
import java.util.List;

public class Persons {
	@SerializedName("start_index")
	public Long startIndex;
	@SerializedName("links")
	public Links links;
	@SerializedName("active_count")
	public Long activeCount;
	@SerializedName("items")
	public List<Person> items = new ArrayList<>();
	@SerializedName("ceased_count")
	public Long ceasedCount;
	@SerializedName("items_per_page")
	public Long itemsPerPage;
	@SerializedName("total_results")
	public Long totalResults;
}

File: app/src/main/java/com/babestudios/companyinfouk/data/model/persons/Persons.kt
package com.babestudios.companyinfouk.data.model.persons


import android.os.Parcelable
import com.google.gson.annotations.SerializedName
import kotlinx.android.parcel.Parcelize

import java.util.ArrayList

@Parcelize
class Persons(
		@SerializedName("start_index")
		var startIndex: Long? = null,
		@SerializedName("links")
		var links: Links? = null,
		@SerializedName("active_count")
		var activeCount: Long? = null,
		@SerializedName("items")
		var items: List<Person> = ArrayList(),
		@SerializedName("ceased_count")
		var ceasedCount: Long? = null,
		@SerializedName("items_per_page")
		var itemsPerPage: Long? = null,
		@SerializedName("total_results")
		var totalResults: Long? = null
) : Parcelable